internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -355,13 +349,14 @@ type external struct {
 	kube   client.Client
 }
 
+func (e *external) Disconnect(ctx context.Context) error {
+	// Unimplemented, required by newer versions of crossplane-runtime
```

Comment:
need to reflect this in the doc schemes but this can be done in a follow up PR too!

Reply:
Umm by schemes do you mean the diagrams under the docs/ dir? Or something else? Do you have editable sources for those images?